base/docs/helpdb/Base.jl

```diff
@@ -7376,7 +7376,7 @@ for the effects of compilation.
     Array(dims)
 
 `Array{T}(dims)` constructs an uninitialized dense array with element type `T`. `dims` may
-be a tuple or a series of integer arguments. The syntax `Array(T, dims)` is also available,
+be a tuple or a series of integer arguments. The syntax `Array{T}(dims)` is also available,
```

Comment:
Okay, I tried, and I have to say, I'm not sure how to do the deprecation. I understand it should be something like
```julia
@deprecate old new
```
but if I try something like
```julia
@deprecate Array{T,N}(::Type{T}, d::NTuple{N,Int}) = Array{T}(d)
```
I get something like (plus a lot more backtracing)
```jula
deprecated.jl
require.jl
error during bootstrap:
LoadError(at "sysimg.jl" line 337: LoadError(at "deprecated.jl" line 1186: Base.MethodError(f=Base.#@deprecate(), args=(Expr(:call, Expr(:curly, :Array, :T, :N)::Any, Expr(:::, Expr(:curly, :Type, :T)::Any)::Any, Expr(:::, :d, Expr(:curly, :NTuple, :N, :Int)::Any)::Any)::Any = Expr(:block, Expr(:line, 1186, :deprecated.jl)::Any, Expr(:call, Expr(:curly, :Array, :T)::Any, :d)::Any)::Any,))))
rec_backtrace at /home/pkm/julia5/julia/src/stackwalk.c:83
record_backtrace at /home/pkm/julia5/julia/src/task.c:228
jl_throw at /home/pkm/julia5/julia/src/task.c:514
eval at /home/pkm/julia5/julia/src/interpreter.c:415
jl_parse_eval_all at /home/pkm/julia5/julia/src/ast.c:773
jl_load at /home/pkm/julia5/julia/src/toplevel.c:583
#218 at ./sysimg.jl:188
jl_call_method_internal at /home/pkm/julia5/julia/src/julia_internal.h:78
do_call at /home/pkm/julia5/julia/src/interpreter.c:58
eval at /home/pkm/julia5/julia/src/interpreter.c:174
jl_toplevel_eval_flex at /home/pkm/julia5/julia/src/toplevel.c:545
jl_toplevel_eval_flex at /home/pkm/julia5/julia/src/toplevel.c:433
jl_toplevel_eval_flex at /home/pkm/julia5/julia/src/toplevel.c:443
jl_parse_eval_all at /home/pkm/julia5/julia/src/ast.c:777
jl_load at /home/pkm/julia5/julia/src/toplevel.c:583
exec_program at /home/pkm/julia5/julia/ui/repl.c:483
true_main at /home/pkm/julia5/julia/ui/repl.c:550
main at /home/pkm/julia5/julia/ui/repl.c:671
unknown function (ip: 0x7f7298be0a3f)
unknown function (ip: 0x401788)
``` 
or if I delete all the methods you linked to
```jlcon
docs/core.jl
inference.jl
A method error occurred before the base module was defined. Aborting...
Array
(Any, 9)
while loading inference.jl, in expression starting on line 3467
rec_backtrace at /home/pkm/julia5/julia/src/stackwalk.c:83
jl_method_error_bare at /home/pkm/julia5/julia/src/gf.c:965
jl_method_error at /home/pkm/julia5/julia/src/gf.c:976
jl_apply_generic at /home/pkm/julia5/julia/src/gf.c:1581
Type at ./inference.jl:92
jl_call_method_internal at /home/pkm/julia5/julia/src/julia_internal.h:78
typeinf_edge at ./inference.jl:1498
jl_call_method_internal at /home/pkm/julia5/julia/src/julia_internal.h:78
typeinf at ./inference.jl:1528
jl_call_method_internal at /home/pkm/julia5/julia/src/julia_internal.h:78
macro expansion; at ./inference.jl:3468
jl_call_method_internal at /home/pkm/julia5/julia/src/julia_internal.h:78
jl_parse_eval_all at /home/pkm/julia5/julia/src/ast.c:777
jl_load at /home/pkm/julia5/julia/src/toplevel.c:583
include at ./boot.jl:222
jl_call_method_internal at /home/pkm/julia5/julia/src/julia_internal.h:78
do_call at /home/pkm/julia5/julia/src/interpreter.c:58
eval at /home/pkm/julia5/julia/src/interpreter.c:174
jl_toplevel_eval_flex at /home/pkm/julia5/julia/src/toplevel.c:545
jl_toplevel_eval_flex at /home/pkm/julia5/julia/src/toplevel.c:433
jl_toplevel_eval_flex at /home/pkm/julia5/julia/src/toplevel.c:443
jl_toplevel_eval_in_warn at /home/pkm/julia5/julia/src/builtins.c:547
eval at ./boot.jl:225
jl_call_method_internal at /home/pkm/julia5/julia/src/julia_internal.h:78
do_call at /home/pkm/julia5/julia/src/interpreter.c:58
eval at /home/pkm/julia5/julia/src/interpreter.c:174
jl_toplevel_eval_flex at /home/pkm/julia5/julia/src/toplevel.c:545
jl_parse_eval_all at /home/pkm/julia5/julia/src/ast.c:777
jl_load at /home/pkm/julia5/julia/src/toplevel.c:583
exec_program at /home/pkm/julia5/julia/ui/repl.c:483
true_main at /home/pkm/julia5/julia/ui/repl.c:550
main at /home/pkm/julia5/julia/ui/repl.c:671
unknown function (ip: 0x7f81f1155a3f)
unknown function (ip: 0x401788)
Allocations: 298087 (Pool: 298048; Big: 39); GC: 0
```
I know I'm doing something wrong, I just can't figure out what.